infer_structcol/tests/test_model.py

```diff
@@ -48,15 +55,16 @@ def test_likelihood():
 
 def test_log_posterior():
     spectrum=Spectrum(500, reflectance = 0.5, sigma_r = 0.1)
-    sample = Sample(500, 200, 200, 1.5, 1)
-
+    sample = Sample(500, 1.5, 1)
+    theta_range = np.array([[0.35,0.74],[70,160],[1,1000]])
+    
     # When parameters are within prior range
-    theta1 = (0.5, 0, 0)
-    post1 = log_posterior(theta1, spectrum, sample, seed=2)
-    assert_approx_equal(post1, -6.329515523909971)
+    theta1 = (0.5, 150, 200, 0, 0)
```

Comment:
Changed it back to theta1 = (0.5, 200, 200, 0, 0), and post1 = -6.329515523909971 when the number of trajectories is 300. The value of post1 also depends on the number of trajectories, so it changes when ntrajectories = 600.